autoagora/main.py

```diff
@@ -95,6 +104,16 @@ async def allocated_subgraph_watcher():
                     logging.info(
                         "Added model update loop for subgraph %s", new_subgraph
                     )
+                if args.multi_root_queries:
+                    # Add the multi root queries (mrq)
+                    update_loops[new_subgraph].model = aio.ensure_future(
```

Comment:
This will override the other model builder. You should add a new element for this in the `SubgraphUpdateLoops` dataclass